pkgs/development/python-modules/safeio/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchpatch }:
+
+buildPythonPackage rec {
+  pname = "safeio";
+  version = "1.2";
```

I suggest to disable new modules on all Python releases which are EOL because the changes are high that nobody tests if the module is working. 

```suggestion
  version = "1.2";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```